gnomad/utils/vep.py

```diff
@@ -85,34 +84,39 @@
 """
 
 
-def vep_context_ht_path(ref: str = "GRCh37"):
-    if ref not in VEP_REFERENCE_DATA.keys():
-        raise DataException(
-            "Select reference as one of: {}".format(",".join(VEP_REFERENCE_DATA.keys()))
-        )
-    return VEP_REFERENCE_DATA[ref]["all_possible"]
+def vep_context_resource(ref: str = "GRCh37") -> VersionedTableResource:
+    """
+    Get VEP context resource for the genome build `ref`
 
+    :param ref: Genome build
+    :return: VEPed context resource
+    """
+    import gnomad.resources.grch37.reference_data as grch37
+    import gnomad.resources.grch38.reference_data as grch38
 
-def vep_config_path(ref: str = "GRCh37"):
-    if ref not in VEP_REFERENCE_DATA.keys():
-        raise DataException(
-            "Select reference as one of: {}".format(",".join(VEP_REFERENCE_DATA.keys()))
-        )
-    return VEP_REFERENCE_DATA[ref]["vep_config"]
+    vep_context_res = grch37.vep_context if ref == "GRCh37" else grch38.vep_context
+    return vep_context_res
 
 
-def vep_or_lookup_vep(ht, reference_vep_ht=None, reference=None, vep_config=None):
+def vep_or_lookup_vep(
+    ht, reference_vep_ht=None, reference=None, vep_config=None, vep_version=None
+):
     """
     VEP a table, or lookup variants in a reference database
 
     :param ht: Input Table
     :param reference_vep_ht: A reference database with VEP annotations (must be in top-level `vep`)
     :param reference: If reference_vep_ht is not specified, find a suitable one in reference (if None, grabs from hl.default_reference)
     :param vep_config: vep_config to pass to hl.vep (if None, a suitable one for `reference` is chosen)
+    :param vep_version: Version of VEPed context Table to use, if none is supplied the default `vep_context` resource will be used
```

Comment:
Could this be updated to match the lines above it "...( if None, the default...)"? Also, not a comment for this line but could you fix L112 to VEPed? It was already in the repo so I cannot comment on it.